src/components/containers/EventCard/EventCard.module.scss

```
@@ -57,3 +57,11 @@
     -webkit-mask-image: -webkit-gradient(linear, 50% top, right top, from(#FFF), to(transparent));
     mask-image: linear-gradient(90deg, #FFF 50%, transparent);
 }
+
+
+.timeToEvent {
+    display: inline-block;
+    padding: 0 4px;
+    background-color: #fff;
```

W większości projektu korzystamy z pełnych kolorków, wiem że to skrót i można to zapisać tak jak to zrobiłeś ale to po prostu kwestia preferencji :D
```suggestion
    background-color: #FFFFFF;
```